client/src/pages/Questions/index.js

```diff
@@ -35,46 +37,82 @@ const  TabPanel = (props) => {
 
 const useStyles = makeStyles(theme => ({
   root: {
-    margin: "0 auto",
-      ['@media (max-width:780px)']: {
-       width: '100%'
-      }
-    },
-    indicator: {
-    height: 0,
-    },
+    margin: '0 auto',
+    ['@media (max-width:780px)']: {
+      width: '100%'
+    }
+  },
+  indicator: {
+    height: 0
+  }
 }))
 
-const  Page4 = (props) => {
-
-  const { questions } = props;
-  const [answer1, setAnswer1] = useState("");
-  const [answer2, setAnswer2] = useState("");
-  const [answer3, setAnswer3] = useState("");
+const Page4 = props => {
+  const { questions } = props
+  const [answer1, setAnswer1] = useState('')
+  const [answer2, setAnswer2] = useState('')
+  const [answer3, setAnswer3] = useState('')
   const classes = useStyles()
-  const theme = useTheme();
-  const [value, setValue] = React.useState(0)
+  const theme = useTheme()
+  const [value, setValue] = useState(0)
+  const [loading, setLoading] = useState(false)
+  const [open, setOpen] = React.useState(false)
 
-  const  handleChange = (event, newValue) => {
+  const handleChange = (event, newValue) => {
     setValue(newValue)
   }
 
-  const  handleChangeIndex = (index) => {
+  const handleChangeIndex = index => {
     setValue(index)
   }
 
-  const  handleSubmit = () => {
-    
-    if(answer1.split(" ")[0] == "" || answer1.split(" ").length > questions[0].wordsLimit ||
-       answer2.split(" ")[0] == "" || answer2.split(" ").length > questions[1].wordsLimit ||
-       answer3.split(" ")[0] == "" || answer3.split(" ").length > questions[2].wordsLimit
+  const handleSubmit = () => {
+    if (
+      answer1.split(' ')[0] === '' ||
+      answer1.split(' ').length > questions[0].wordsLimit ||
+      answer2.split(' ')[0] === '' ||
+      answer2.split(' ').length > questions[1].wordsLimit ||
+      answer3.split(' ')[0] === '' ||
+      answer3.split(' ').length > questions[2].wordsLimit
     ) {
       Swal.fire({
         type: 'error',
         title: 'Oops...',
-        text: 'Make sure you answered all questions without exceeding word limits!',
+        text:
+          'Make sure you answered all questions without exceeding word limits!'
       })
     }
+
+    setLoading(true)
+    axios({
```

Comment:
not super familiar with axios so if there is anything I've handled wrong do tell